src/Mime.php

```diff
@@ -125,21 +125,28 @@ public static function encodeQuotedPrintable(
                 $ptr = $lineLength;
             }
 
-            // Try to prevent that the first character of a line is a dot
-            // Outlook Bug: http://engineering.como.com/ghost-vs-outlook/
-            while ($ptr > 1 && $ptr < strlen($str) && $str[$ptr] === '.') {
-                --$ptr;
-            }
-
             // Ensure we are not splitting across an encoded character
             $pos = strrpos(substr($str, 0, $ptr), '=');
             if ($pos !== false && $pos >= $ptr - 2) {
                 $ptr = $pos;
             }
 
-            // Check if there is a space at the end of the line and rewind
-            if ($ptr > 0 && $str[$ptr - 1] == ' ') {
-                --$ptr;
+            switch (ord($str[0])) {
+                case 0x2E:
```

Comment:
would be easier to read if you add an comment with the translation of the hex code